python/ray/serve/scripts.py

```diff
@@ -73,3 +88,28 @@ def start(http_host, http_port, http_location, checkpoint_path):
 def shutdown():
     serve.api._connect()
     serve.shutdown()
+
+
+@cli.command(
+    help="""
+[Experimental]
+Create a deployment in running Serve instance. The required argument is the
+import path for the deployment: ``my_module.sub_module.file.MyClass``. The
+class should already be decorated with ``@serve.deployment``.
+""",
+    hidden=True,
+)
+@click.argument("deployment")
+@click.option(
+    "--options-json",
+    default=r"{}",
+    required=False,
+    type=str,
+    help="JSON string for the deployments options")
+def deploy(deployment: str, options_json: str):
+    deployment_cls = import_attr(deployment)
+    if not isinstance(deployment_cls, Deployment):
+        raise ValueError(
+            f"{deployment} is not decorated with @serve.deployment.")
```

Comment:
ah, let's please also support normal classes/functions by calling serve.deployment(deployment_cls) here.

Reply:
and add a test!